modules/light-clients/08-wasm/keeper/keeper_test.go

```diff
@@ -3,7 +3,7 @@ package keeper_test
 import (
 	"testing"
 
-	"github.com/stretchr/testify/suite"
+	testifysuite "github.com/stretchr/testify/suite"
```

Comment:
correct me if I'm wrong, but was this aliased before `testsuite` someplace else or am I mistaken?

Reply:
yea, slightly confused me too tbh, `testsuite` is another package in the e2e module. In that big linter PR, we went with `testifysuite` to try to differentiate from that one too.

Reply:
okay sounds good to me!